eng/globalconfigs/Common.globalconfig

```diff
@@ -0,0 +1,121 @@
+is_global = true
+
+# Default severity for all IDE code style rules with category 'Style'
+dotnet_analyzer_diagnostic.category-Style.severity = warning
+
+# Default severity for all IDE code quality rules with category 'CodeQuality'
+dotnet_analyzer_diagnostic.category-CodeQuality.severity = warning
+
+### Configuration for RSxxxx rules ###
+
+# RS1024: Compare symbols correctly
+# https://github.com/dotnet/roslyn-analyzers/issues/3389
+dotnet_diagnostic.RS1024.severity = none
+
+# Parts exported with MEFv2 must be marked with 'SharedAttribute', un-needed here
+dotnet_diagnostic.RS0023.severity = none
+
+# Exported parts should be marked with 'ImportingConstructorAttribute', un-needed here
+dotnet_diagnostic.RS0034.severity = none
+
+# Use 'null' instead of 'default' for nullable types
+dotnet_diagnostic.RS0038.severity = none
+
+# TODO: Remove this exclusion and fix related issues, tracked by https://github.com/dotnet/roslyn-analyzers/issues/3915
+#dotnet_diagnostic.RS0041.severity = none
+
+### Configuration for IDE code style by diagnostic IDs ###
+
+# IDE0078: Use pattern matching
+# https://github.com/dotnet/roslyn/issues/51691
+# https://github.com/dotnet/roslyn/issues/51693
+dotnet_diagnostic.IDE0078.severity = silent
```

Comment:
Both these issues are fixed back in April. Are we still on a very old CodeStyle package?